pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/LinuxInfoUtils.java

```diff
@@ -254,7 +254,7 @@ public double convertBy(double usageBytes) {
 
     @AllArgsConstructor
     public enum UsageUnit {
-        Kbps(8 / 1024);
+        Kbps(1024);
```

Comment:
Is this unit related?

Reply:
Yes, `tx_bytes` and `rx_bytes`  unit is bytes, not bite. shouldn't divide 8